index.html

```diff
@@ -174,8 +181,8 @@ <h3 class="article__title article__title--sidebar">
           <div class="article">
             <img
               class="article__image"
-              src="https://raw.githubusercontent.com/Code-Your-Future/syllabus/master/html-css/assets/news-cycle.jpg"
-              alt=""
+              src="../images/edinburgh-damascus_thumbnail.png"
+              alt="edinburgh-damascus"
```

Comment:
Here, images is in the same directory as index.html so it doesn't need the `../`

Reply:
done, thanks